package.json

```diff
@@ -14,6 +14,7 @@
   "files": [
     "package.json",
     "use-force-update.d.ts",
+    "use-force-update.js.map",
```

Comment:
I'm not sure the map file is useful here, since the `src` directory is not included.

Reply:
sorry, totally misunderstood, will push again without it (: